cesium/internal/index/domain.go

```diff
@@ -98,7 +114,7 @@ func (i *Domain) Distance(
 	domainBounds.Lower = iter.Position()
 
 	for {
-		if !iter.Next() {
+		if !iter.Next() || (continuous && !effectiveDomainBounds.ContainsRange(iter.TimeRange())) {
```

Comment:
I think there could be a clearer way to write out this control flow.

Reply:
Hm, I've tried a few things out but I still think this is the most understandable:
Some things I tried are:
- `if !iter.Next() || !effectiveDomainBounds.ContainsRange(iter.TimeRange()` -> this requires another check since we don't care if the second statement is satisfied if `continuous = false`
- factoring out `if continuous`, but this would need an else to handle the case where iterator is exhausted.

Admittedly the `continuous` being in two lines does weird me out. LMK if you have any ideas

Reply:
i think youre right it won't be much clearer any other way